examples/darwin-framework-tool/commands/common/CHIPCommandBridge.h

```diff
@@ -52,6 +52,9 @@ class CHIPCommandBridge : public Command {
         AddArgument("commissioner-vendor-id", 0, UINT16_MAX, &mCommissionerVendorId,
             "The vendor id to use for darwin-framework-tool. If not provided, chip::VendorId::TestVendor1 (65521, 0xFFF1) will be "
             "used.");
+        AddArgument("use-xpc", 0, 1, &mUseXPC, "This option uses the XPC implementation.");
+        AddArgument("use-xpc-service-name", &mUseXPCServiceName, "The name of the XPC service to connect to.");
+        AddArgument("use-xpc-controller-id", &mUseXPCControllerId, "The id of the XPC controller to connect with.");
```

Comment:
How is someone supposed to pick those strings?  Is just using a unique string enough?  Does it need to be the same unique string across multiple commands?  This part is extremely unclear.

Reply:
Let's see  if  #36786 clarifies this issue.